pytraj/all_actions.py

```diff
@@ -1927,9 +1937,10 @@ def rmsd(traj=None,
 
     """
 
-    nofit_ = ' nofit ' if nofit else ''
-    mass_ = ' mass ' if mass else ''
-    options = nofit_ + mass_
+    nofit_ = 'nofit' if nofit else ''
+    mass_ = 'mass' if mass else ''
+    nomod_ = 'nomod' if not update_coordinate else ''
+    options = ' '.join((nofit_ + mass_, nomod_))
```

Comment:
Shouldn't this be `(nofit_, mass_, nomod_)`?  As is, this could be rendered to `nofitmass nomod' if all 3 are specified.